packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4742,3 +4743,31 @@ export class Gitxnas extends Gtxnas {
 		super.execute(stack);
 	}
 }
+
+/**
+ * Takes the last value from stack and if base64encoded, decodes it and push it back to the stack,
+ * othervise throws an error
+ * https://github.com/algorand/go-algorand/pull/3220
+ */
+export class Base64Decode extends Op {
+	readonly line: number;
+	/**
+	 * Asserts 0 arguments are passed.
+	 * @param args Expected arguments: [] // none
+	 * @param line line number in TEAL file
+	 */
+	constructor(args: string[], line: number) {
```

Comment:
Opcode is `base64_decode e`, so we should have args = [e]. 
e  = {0, 1}.
| Index | Name | Notes |
| - | ------ | --------- |
| 0 | URLEncoding |  |
| 1 | StdEncoding |  |

More details [here](https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/TEAL_opcodes.md#base64_decode-e)